src/renderers/shared/fiber/__tests__/__snapshots__/ReactIncrementalPerf-test.js.snap

```diff
@@ -6,10 +6,11 @@ exports[`ReactDebugFiberPerf captures all lifecycles 1`] = `
   ⚛ AllLifecycles [mount]
     ⚛ AllLifecycles.componentWillMount
     ⚛ AllLifecycles.getChildContext
-  ⚛ (Committing Changes)
-    ⚛ (Committing Host Effects: 1 Total)
-    ⚛ (Calling Lifecycle Methods: 1 Total)
-      ⚛ AllLifecycles.componentDidMount
+
+⚛ (Committing Changes)
```

Comment:
AFAICT the changes here are all because the commit phase is now a separate entry point.